### PR TITLE
feat: support file:///path/to/secret/file as SSL private key passphrase

### PR DESCRIPTION
**Pull request by zmstone**
Release version: v/e5.8.2

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [x] Added tests for the changes
- [ ] Added property-based tests for code which performs user input validation
- [x] Changed lines covered in coverage report
- [x] Change log has been added to `changes/(ce|ee)/(feat|perf|fix|breaking)-<PR-id>.en.md` files
- [ ] For internal contributor: there is a jira ticket to track this change
- [ ] Created PR to [emqx-docs](https://github.com/emqx/emqx-docs) if documentation update is required, or link to a follow-up jira ticket
- [ ] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

File: apps/emqx/src/emqx_tls_lib.erl [hunk 1]
```diff
@@ -576,7 +578,7 @@ to_client_opts(Type, Opts) ->
                     {ciphers, Ciphers},
                     {reuse_sessions, Get(reuse_sessions)},
                     {depth, Get(depth)},
-                    {password, ensure_str(Get(password))},
+                    {password, ensure_password(Get(password))},
```
Comment (keynslug) [inline apps/emqx/src/emqx_tls_lib.erl:581]: Q: Why not mark it as `emqx_schema_secret:secret()` on the schema level, and then call simple `unwrap/1` here?

Comment (zmstone) [reply apps/emqx/src/emqx_tls_lib.erl:581]: fixed

**Review by keynslug — DISMISSED** [review]
🚀

**Review by keynslug — APPROVED** [review]

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-4875)

**Pull request merged**